### PR TITLE
SetCursor CefCursorType parameter

### PR DESCRIPTION
**Pull request by cuddlyogre**
SetCursor now receives the cursor type. The enums for the types come directly from CEF.

### CODE REVIEW

File: CefSharp.Core/Internals/RenderClientAdapter.h [hunk 1]
```diff
@@ -174,7 +174,7 @@ namespace CefSharp
             virtual DECL void OnCursorChange(CefRefPtr<CefBrowser> browser, CefCursorHandle cursor, CursorType type,
                 const CefCursorInfo& custom_cursor_info) OVERRIDE
             {
-                _renderWebBrowser->SetCursor((IntPtr)cursor);
+				_renderWebBrowser->SetCursor((IntPtr)cursor, (CefSharp::CefCursorType)type);
```
Comment (jankurianski) [inline CefSharp.Core/Internals/RenderClientAdapter.h]: Replace tabs with spaces.

File: CefSharp.Wpf/ChromiumWebBrowser.cs [hunk 2]
```diff
@@ -293,9 +293,9 @@ void IRenderWebBrowser.SetPopupSizeAndPosition(int width, int height, int x, int
         void IRenderWebBrowser.SetPopupIsOpen(bool isOpen)
         {
             UiThreadRunAsync(() => { popup.IsOpen = isOpen; });
-        }
-
-        void IRenderWebBrowser.SetCursor(IntPtr handle)
+        }  
+
```
Comment (jankurianski) [inline CefSharp.Wpf/ChromiumWebBrowser.cs]: Unnecessary addition of whitespace.

File: CefSharp/CefSharp.csproj [hunk 3]
```diff
@@ -93,6 +93,7 @@
     <Compile Include="DefaultResourceHandlerFactory.cs" />
     <Compile Include="DependencyChecker.cs" />
     <Compile Include="IJavascriptCallback.cs" />
+    <Compile Include="CEFCursorType.cs" />
```
Comment (jankurianski) [inline CefSharp/CefSharp.csproj]: File name doesn't seem consistent with class name.  Shouldn't it be `CefCursorType.cs`?

File: CefSharp3.sln.DotSettings [hunk 4]
```diff
@@ -98,6 +98,7 @@
 	<s:Boolean x:Key="/Default/Environment/SearchAndNavigation/GotoSingleUsageImmediately/@EntryValue">False</s:Boolean>
 	<s:Boolean x:Key="/Default/Environment/SearchAndNavigation/MergeOccurences/@EntryValue">True</s:Boolean>
 	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ESettingsUpgrade_002EMigrateBlankLinesAroundFieldToBlankLinesAroundProperty/@EntryIndexedValue">True</s:Boolean>
+	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EReSharper_002EPsi_002ECSharp_002ECodeStyle_002ESettingsUpgrade_002EMigrateThisQualifierSettings/@EntryIndexedValue">True</s:Boolean>
```
Comment (jankurianski) [inline CefSharp3.sln.DotSettings]: Could you remove this from the PR?  If we do change the ReSharper settings, this should be a separate PR rather than sneak in this change :smile:

**Comment by jankurianski** [comment]
Thanks for the PR @cuddlyogre :+1: 

Just a few formatting and commit cleaning issues to sort out.

**Comment by jankurianski** [comment]
Oh and when you do clean up the issues, remember to leave a new comment here.  This ensures the project maintainers get an email notification.  We don't get told about further commits alone.

**Comment by cuddlyogre** [comment]
I have made the requested changes.

File: CefSharp.Core/Internals/RenderClientAdapter.h (hunk 1 above)
Comment (cuddlyogre) [inline CefSharp.Core/Internals/RenderClientAdapter.h]: I've been using Intellij for so long. I had forgotten that the tab key inserts a tab character sometimes.

File: CefSharp/CEFCursorType.cs [hunk 5]
```diff
@@ -0,0 +1,54 @@
+﻿// Copyright © 2010-2015 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+namespace CefSharp
+{
+    public enum CefCursorType
+    {
+        CT_POINTER = 0,
```
Comment (amaitland) [inline CefSharp/CEFCursorType.cs]: Can you remove the `CT_` prefix and make these proper case. Bring them inline with `.Net` styling.
e.g. `CT_POINTER` becomes `Pointer`

**Comment by cuddlyogre** [comment]
The changes to the CefCursorType names has been made.

**Comment by amaitland** [comment]
Great, thanks :+1:

**Comment by cuddlyogre** [comment]
You're very welcome!
